debify.gemspec

```diff
@@ -19,7 +19,8 @@ Gem::Specification.new do |spec|
   
   spec.add_dependency "gli"
   spec.add_dependency "docker-api", "~> 1.33"
-  spec.add_dependency "conjur-cli"
+  spec.add_dependency "conjur-cli" , "~> 5"
```

Comment:
what is the CLI used for? pretty big dependency, is all.

Reply:
debify pulls in the CLI so it can use the user's credentials for to conjurop, and use the API to get the Artifactory username and password.

Do you remember why you (I think it was you) did it this way, rather than using summon to pass environment variables? Having debify use the CLI is actually causing me trouble right now, and it would be easier if those creds were pass through the environment?

Reply:
I think I pulled it in so we wouldn't have to chain CLIs together. A lot of our pipelines just say `debify command` right now, so if we wanted to prefix w/ summon we'd have to write a lot of code.

We could install https://github.com/JFrogDev/artifactory-cli-go on jenkins executors, since artifactory is a big part of our CI cycle and it's a binary (easy to install). Then `debify` doesn't need to do publishing at all. What do you think of that?

We could always shadow `debify publish` so we don't have to update all of our pipelines.